src/sidebar/Sidebar.jsx

```diff
@@ -15,7 +15,7 @@ export default function Sidebar(){
   return (
     <div className={!isAuth ? "hidden" : ""}>
```

Comment:
@arkumari2000 I don't think this is the right way to do conditional rendering. Anyone can change the toggle the `display:none` css property from the developer tools. Ideally, if `isAuth` is false, then the component should return null. So please don't use this `hidden` class as a way of conditional rendering a component.
cc: @mtreacy002

Reply:
I agree @Rahulm2310 , @arkumari2000 . Perhaps the reason why you used `hidden` is because you didn't run the backend servers in parallel as you're supposed to and use this to by pass the `isAuth`. For this time, we can let it slide since the manual tests shows that it works through running backend servers. But for it to be merged to the upstream branch, can you please remove the `hidden` option as per @Rahulm2310 suggested? Thanks

Reply:
Yes sure, I will do the conditional rendering and not hidden property.